job-dsls/jobs/kie/kogito/kogito_docs_upload.groovy

```diff
@@ -38,11 +38,11 @@ pipeline {
         }
         stage ('checkout kie-docs') {
             steps {
-                checkout([$class: 'GitSCM', branches: [[name: 'master-kogito']], browser: [$class: 'GithubWeb', repoUrl: 'git@github.com:kiegroup/kie-docs.git'], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'kie-docs']], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'kie-ci-user-key', url: 'git@github.com:kiegroup/kie-docs.git']]])
+                checkout([$class: 'GitSCM', branches: [[name: 'main-kogito']], browser: [$class: 'GithubWeb', repoUrl: 'git@github.com:kiegroup/kie-docs.git'], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'kie-docs']], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'kie-ci-user-key', url: 'git@github.com:kiegroup/kie-docs.git']]])
```

Comment:
this requires https://github.com/kiegroup/kie-docs/blob/d7ed223857d1e18588c76802eb765137140123ed/.github/workflows/push.yml#L7 to be updated

Reply:
@Ginxo good spot:
https://github.com/kiegroup/kie-docs/pull/3806